dev-env/bootstrap/clusters.yaml

```diff
@@ -0,0 +1,63 @@
TlFVMUNRV1k0ZDBoUldVUldVakJQUWtKWlJVWk1RVFE1VjJNd1JsZG1TbTVaTjFRMEwzcG1VWGxqYzNGaVYyeE5RVEJIUTFOeFIxTkpZak1LUkZGRlFrTjNWVUZCTkVsQ1FWRkJOVGh2UVZWVlUweGxha0ZGUzBKTGJuTkZSelE0TlVSU1RHNW1TMFJuYVRkc1oxTkhkMWwzWXpVM01YRk1NWEl3UlFwT2RVUlJRU3QyUVZWTU9FZHVkMnBoTDJJNVYwTm9ZM0ZxTlZJelRqbDNTVzVSTW1FelZWQjZiRkpyTjJZMFpGTnlkWFJNYmtrM1JWaE9OMDR2UVhSQkNtVXJkekJrYjJsS1JtTkpWM05vTjJ0Tkx6aDNiREJLVEM4NU0xbHZhbVpRVDNRclltdExNME42VTFOVVMwbGtiVVZ5VEhwVlRUVlVka0ZUWlRoMk5VWUtOekE0UlhwcFNUaHRaVzVsYkZGSVIwMXRSVTlpYTFBMmRteDZaVzR6VEdwNlMxaFROR2xDU25SQ1J6TXlaMHgwZFRKUmIzWk9Xa2R1WXpKcmNEYzRZd3BGTTNBNE9IRnRaazgyUkRWbE5scHRURVJ0TVRkd2JuUTBXR2x6V0ZsaEwxSmtVRlJUWW5SeE16TnpNRkJ0U2pkRWRYY3labWt6TDIxTVFYTnFWMlJIQ21KRmIyRmtUbGhFT0M5TVZqWmlNV1p0WjJFMGFFUkJLMGhKYW1OeE5EZHdOVGhVZWdvdExTMHRMVVZPUkNCRFJWSlVTVVpKUTBGVVJTMHRMUzB0Q2c9PQogICAgc2VydmVyOiBodHRwczovL2FwaS5xYS5ncmVlbmhvdXNlLnNob290LmNhbmFyeS5rOHMtaGFuYS5vbmRlbWFuZC5jb20KICBuYW1lOiBrdWJlY3RsLXN5bmM6Mzk1OmdyZWVuaG91c2UtcWE6MS4yNi40CmNvbnRleHRzOgotIGNvbnRleHQ6CiAgICBjbHVzdGVyOiBrdWJlY3RsLXN5bmM6Mzk1OmdyZWVuaG91c2UtcWE6MS4yNi40CiAgICB1c2VyOiBrdWJlY3RsLXN5bmM6Mzk1Om9pZGNAZ3JlZW5ob3VzZS1xYQogIG5hbWU6IGdyZWVuaG91c2UtcWEKY3VycmVudC1jb250ZXh0OiBncmVlbmhvdXNlLXFhCnVzZXJzOgotIG5hbWU6IGt1YmVjdGwtc3luYzozOTU6b2lkY0BncmVlbmhvdXNlLXFhCiAgdXNlcjoKICAgIGF1dGgtcHJvdmlkZXI6CiAgICAgIGNvbmZpZzoKICAgICAgICBjbGllbnQtaWQ6IGdyZWVuaG91c2UKICAgICAgICBjbGllbnQtc2VjcmV0OiAiIgogICAgICAgIGV4dHJhLXNjb3BlczogcHJvZmlsZSxlbWFpbCxncm91cHMsb2ZmbGluZV9hY2Nlc3MKICAgICAgICBpZC10b2tlbjogZXlKaGJHY2lPaUpTVXpJMU5pSXNJbXRwWkNJNklqSXpNalprTnpNd01HUmxZMkV6TjJRM05qZGpNR0V4TnpFelpUa3hNak5oTW1KbU0ySXdOalVpZlEuZXlKcGMzTWlPaUpvZEhSd2N6b3ZMMkYxZEdndVozSmxaVzVvYjNWelpTMXhZUzVsZFMxdWJDMHhMbU5zYjNWa0xuTmhjQ0lzSW5OMVlpSTZJa05uWkVwT1ZHTXdUbXBGTUVWbmNHNWpiVlpzWW0xb2RtUllUbXdpTENKaGRXUWlPaUpuY21WbGJtaHZkWE5sSWl3aVpYaHdJam94TmpnMk5UVTVOakU0TENKcFlYUWlPakUyT0RZMU5UWXdNVGdzSW1GMFgyaGhjMmdpT2lKeVZrZGtWekkwZGpobk9ISTFjRmhJUnpJMFZFbDNJaXdpWTE5b1lYTm9Jam9pU0VWaVozaExUMk5aYzFKcVRqUXpibWc1ZG1sSlFTSXNJbVZ0WVdsc0lqb2lkWGRsTG0xaGVXVnlNREZBYzJGd0xtTnZiU0lzSW1WdFlXbHNYM1psY21sbWFXVmtJanAwY25WbExDSm5jbTkxY0hNaU9sc2liM0puWVc1cGVtRjBhVzl1T21keVpXVnVhRzkxYzJVaUxDSjBaV0Z0T21Ga2JXbHVJaXdpY205c1pUcG5jbVZsYm1odmRYTmxPbUZrYldsdUlsMHNJbTVoYldVaU9pSkpOVGMwTmpFMElpd2ljSEpsWm1WeWNtVmtYM1Z6WlhKdVlXMWxJam9pVlhkbElFMWhlV1Z5SW4wLlI4d0d4VTNWS0VVaEp5UkZSTWdrbEp0YVFJRmphcjFlanBETlZheHJZdzk5Um9jOHFqakt0MEQ4cmVGWV9OSkw2cHZKZW9sVmFmZjd0WGZ3UHJCSkpuVFBBM05LRFBDRGVvOVNsS2VZQl9xVDF2b1hPQUlHYjF1RWJJY2Vhb19UWFBpbnRuYmRNejc4ekNiOGphOGZwaGxzbVVZZWRGOVZXRW5xS252X09Ocmp0SUhtQzJCbjR1cXA2NlZ2YVhva1JINkE4THBHNUVLMGgyYmJQenZHLW1XT255T3hmdlRYcjN6bmNGQlpyLTFreFZIeWRIMzJCRmltb1lQYnQtdGozV2lCU3QwZHNhNzJ5UXlncXpxWlFWeUwzUTZ3VkNsMkdMajZBRlZWdlNuQmg1bGdkYVFzZzJHbnlsSnQ5TDdMOF9xY2k2UW02cUFWVklTVXRoVVhYdwogICAgICAgIGlkcC1pc3N1ZXItdXJsOiBodHRwczovL2F1dGguZ3JlZW5ob3VzZS1xYS5ldS1ubC0xLmNsb3VkLnNhcAogICAgICAgIHJlZnJlc2gtdG9rZW46IENobDRaM2QzWm5kMk5XdHFkelJoWW10cWFHSmxhemQyWm5oekVobHZiV2R3YW1WaVlqVnNlV1J3Tld0dE5EZG1jbTVrYXpONQogICAgICBuYW1lOiBvaWRjCg==
```

Comment:
This is a kubeconfig for our internal qa cluster. Since this won't work anyhow can we replace with some Kind cluster config or similar?

Reply:
replaced with complete dummy data.

Thanks for catching this.